plutus-playground-client/src/MainFrame.purs

```diff
@@ -461,7 +461,7 @@ render state@(State {currentView})  =
             ]
         ]
     , viewContainer currentView Editor $
-        [ editorPane state
+        [ editorPane (Map.lookup "Vesting" StaticData.demoFiles) (view _compilationResult state)
```

Comment:
If you fancy it, `where`-clausing this the way you have with Meadow would be nice.